src/api.js

```diff
@@ -24,4 +24,15 @@ export const fetchReviewById = (reviewId) => {
             return (err)
         }
     )
+}
+
+export const fetchCommentsForReviewId = (reviewId) => {
+    return ConsApi
+        .get(`/reviews/${reviewId}/comments`)
+        .then(({ data }) => {
+            return data.comments;
+        }).catch(err => {
```

Comment:
Catch blocks are unnecessary here as our back end should be handling these errors. We have not covered front end error handling. Please remove any catch blocks in your api file.